signs-work-order-map/utils/mapUtils.tsx

```diff
@@ -0,0 +1,121 @@
+import { useMemo } from "react";
+import bbox from "@turf/bbox";
+import { lineString } from "@turf/helpers";
+import { LngLatBoundsLike } from "mapbox-gl";
+import { Marker } from "react-map-gl/mapbox";
+import { Sign, KnackToIFrameMessage, LatLon } from "@/types/map";
+
+/**
+ * Takes array of Signs and if signs exist, returns bounding box for signs
+ * @param signs
+ * @returns bbox extent in [minX, minY, maxX, maxY] order or undefined
+ */
+export const formatBounds = (signs: Sign[]): undefined | LngLatBoundsLike =>
```

Comment:
It might be nice to add `use` to the start of the custom hook functions in this file. I really love seeing how all this data is build up btw! 🚀